### PR TITLE
26 naming to organization

### PR DESCRIPTION
**Pull request by philwilliammee**
Renamed colleges to organizations and reset composer autoload.

added unit test for db create Organization. I couldnt get route testing to work.

### CODE REVIEW

**Comment by philwilliammee** [comment]
this change touches a lot of files

File: resources/views/son/form.blade.php [hunk 1]
```diff
@@ -124,35 +124,35 @@ class="form-control @error('submiting_for_netid') is-invalid @enderror"
                 </div>
             </div>
 
-            <!-- College Unit -->
+            <!-- Organization Unit -->
```
Comment (woodseowl) [inline resources/views/son/form.blade.php:127]:
```suggestion
            <!-- Organization -->
```

File: resources/views/son/edit.blade.php [hunk 2]
```diff
@@ -137,38 +137,38 @@ class="form-control @error('submiting_for_netid') is-invalid @enderror"
 
                             </div>
 
-                            <!-- College Unit -->
+                            <!-- Organization Unit -->
```
Comment (woodseowl) [inline resources/views/son/edit.blade.php:140]:
```suggestion
                            <!-- Organization -->
```

File: resources/views/sonFormFields/edit.blade.php [hunk 3]
```diff
@@ -329,50 +329,50 @@ class="form-control @error('itsg_name_help_text') is-invalid @enderror"
...
-                        <!-- college_name label -->
+                        <!-- organization_name label -->
                         <div class="form-group row mt-4">
                             <label
-                                for="college_name_label"
+                                for="organization_name_label"
                                 class="col-md-4 col-form-label text-md-right required"
                             >
-                                {{ __('College/Unit Name Label') }}
+                                {{ __('Organization/Unit Name Label') }}
```
Comment (woodseowl) [inline resources/views/sonFormFields/edit.blade.php:338]:
```suggestion
                                {{ __('Organization Name Label') }}
```

**Pull request merged**